hilti/src/compiler/clang.cc

```diff
@@ -633,6 +633,11 @@ Result<Library> ClangJIT::Implementation::compileModule(llvm::Module&& module) {
                                      "-o",
                                      *library_path};
 
+    const auto& ld_flags = context->options().optimize ? hilti::configuration().runtime_ld_flags_release :
+                                                         hilti::configuration().runtime_cxx_flags_debug;
```

Comment:
```suggestion
    const auto& ld_flags = context->options().optimize ? hilti::configuration().runtime_ld_flags_release :
                                                         hilti::configuration().runtime_ld_flags_debug;
```

Typo, right?

Reply:
Yes, but actually the other way around. We'd need `runtime_cxx_flags_release` here here to trigger the correct link when using `-fprofile-instr-generate -fcoverage-mapping`.

I am removing this patch from this patchset for now since I am not 100% clear on what this could mean for other arbitrary CXX flags users could pass. A more controlled approach for properly including coverage information might be an extra configure time switch which sets correct compiler _and_ linker flags.